src/chainbin/chain-make-num-fst-e2e.cc

```diff
@@ -0,0 +1,121 @@
+// chainbin/chain-make-num-fst-e2e.cc
+
+// Copyright 2019  Johns Hopkins University (author: Yiwen Shao)
```

Comment:
Just put Yiwen Shao as the author as you are not an employee (or if you have a part-time job with Hopkins, at least you are not doing this in that status)